src/commands/init/spec/init.spec.server.ts

```diff
@@ -24,6 +25,7 @@ describe('sasjs init', () => {
     const appName = `test-app-init-${generateTimestamp()}`
 
     process.projectDir = path.join(__dirname, appName)
+    await setConstants()
```

Comment:
should be moved to `beforeEach` function

Reply:
These are in right place, `process.sasjsConstants` are according to `process.projectDir`